### ISSUE TITLE
Custom tags missing in some contexts 

### ISSUE THREAD

**Issue by Sandeep-Singh-Spartan** [getsentry/sentry-docs#7005]
### Steps to Reproduce

I'm trying to use the Sentry package in a laravel application. but I am facing some inconsistency in the custom tags some time they appear in the sentry dashboard sometimes not.  I am attaching the code snippet .

  if (app()->bound('sentry') && $this->shouldReport($e)) {
            app('sentry')->configureScope(function (\Sentry\State\Scope $scope) use ($e): void {
                $scope->setTag('tracking_code', $this->trackingCode);

                if (auth()->check()) {
                    $scope->setUser($this->getUser());
                    $scope->setTag('user_details', json_encode($this->getUser()));
                }
                app('sentry')->captureException($e);
            });
        }

### Expected Result

I am attaching the sentry Dashboard’s screenshot
<img width="842" alt="Screenshot 2023-05-24 at 12 00 13 AM" src="https://github.com/getsentry/sentry-docs/assets/128671203/e79531cc-0b6b-4ad0-ab3c-6ed675ee6e18">
 where I am getting the custom tag 

### Actual Result

some time I am getting the custom tag in sentry dashboard
<img width="844" alt="Screenshot 2023-05-24 at 12 00 28 AM" src="https://github.com/getsentry/sentry-docs/assets/128671203/3d0e9357-8f48-4c04-b089-309e20e51979">

**Comment by shanamatthews** [comment]
Doesn't seem like a docs issue, should probably be moved to Laravel SDK repo or sentry repo

**Comment by Sandeep-Singh-Spartan** [comment]
Yes, I am facing the inconsistent behavior of customer tag **'tracking_code'**, **user_details**. This issue does not seem to be related to the documentation, but rather an issue within the Laravel SDK or possibly another component.
